runtimes/node-17.0/server.js

```diff
@@ -17,9 +17,14 @@ const server = micro(async (req, res) => {
         payload: body.payload ?? '',
     };
 
+    console.stdlog = console.log.bind(console);
+    logs = [];
+    console.log = function(){
+        logs.push(Array.from(arguments));
```

Comment:
Soo if I low 2 things, are they part of one array, or are they separated?

`console.log(1,3)` is `[1,3]` or [[1,3]]`? Also not sure which one is better.

Reply:
if I don't do `Array.from(arguments)` `console.log(1,3)` becomes `{0: 1, 1: 3}`